cmd/packer-sdc/internal/plugincheck/cmd.go

```diff
@@ -0,0 +1,111 @@
+package plugincheck
+
+import (
+	_ "embed"
+	"encoding/json"
+	"fmt"
+	"log"
+	"os/exec"
+	"strings"
+
+	"github.com/pkg/errors"
+)
+
+var (
+	//go:embed README.md
+	readme string
+)
+
+type Command struct {
+}
+
+func (cmd *Command) Help() string {
+	return "\n" + readme
+}
+
+func (cmd *Command) Run(args []string) int {
+	if err := cmd.run(args); err != nil {
+		log.Printf("%v", err)
+		return 1
+	}
+	return 0
+}
+
+func (cmd *Command) run(args []string) error {
+
+	if len(args) != 1 {
+		cmd.Help()
+		return errors.New("plugincheck takes on plugin binary name as argument.\n" +
+			"ex: 'packer-plugin-happycloud'. Check will be run on the binary.")
+	}
+
+	pluginName := args[0]
+
+	if isOldPlugin(pluginName) {
+		fmt.Printf("\n[WARNING] Plugin is named with old prefix `packer-[builder|provisioner|post-processor]-{name})`. " +
+			"These will be detected but Packer cannot install them automatically. " +
+			"The plugin must be a multi-component plugin named packer-plugin-{name} to be installable through the `packer init` command.\n" +
+			"See docs at: https://www.packer.io/docs/plugins.\n")
+		return nil
+	}
+
+	if err := checkPluginName(pluginName); err != nil {
+		return err
+	}
+
+	output, err := exec.Command(pluginName, "describe").Output()
+	if err != nil {
+		return errors.Wrap(err, "failed to describe plugin")
```

Comment:
What do you think about adding an example to this error message? 
```suggestion
                s := fmt.Sprintf("failed to describe plugin using '%s describe'", pluginName)
		return errors.Wrap(err, s, )
```